ObjectiveGit/GTRepository+Status.m

```diff
@@ -125,4 +125,10 @@ - (BOOL)shouldFileBeIgnored:(NSURL *)fileURL success:(BOOL *)success error:(NSEr
 	return (ignoreState == 1 ? YES : NO);
 }
 
+- (GTFileIgnoreState)shouldIgnoreFileURL:(NSURL *)fileURL error:(NSError **)error {
+	BOOL success = false;
```

Comment:
Ha; guess it shows that I've been coding nothing but Swift for the past while...